toolkit/docs/quick_start/quickstart.md

```diff
@@ -0,0 +1,128 @@
+# Quick Start Guide
+  - [Install Prerequisites](#install-prerequisites)
+  - [Clone CBL-Mariner](#Clone-CBL-Mariner)
+  - [Build and Boot an Image](#build-and-boot-an-image)
+    - [VHDX and VHD Images](#vhdx-and-vhd-images)
+    - [ISO Image](#iso-image)
+  
+## **Install Prerequisites**
+Install prerequisites [here](../building/prerequisites.md).
+
+ 
+## **Clone CBL-Mariner**
+From a bash terminal window, clone the CBL-Mariner Repository and check-out a stable build.
+
+```bash
+# Clone the CBL-Mariner repo
+git clone git@github.com:microsoft/CBL-Mariner.git
```

Comment:
>clone git@github.com:microsoft/CBL-Mariner.git [](start = 4, length = 46)

Suggestion: to keep the examples shorter we could also just run `git clone --branch 1.0-stable git@github.com:microsoft/CBL-Mariner.git` and drop the other commands.

Reply:
agree....but I will leave this one as is.  I think it's a little more clear to separate cloning from branch selection in the tutorial...even though most everyone probably gets it.